ZenPacks/zenoss/LinuxMonitor/patches/patches.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+from Products.ZenUtils.Utils import monkeypatch
+from Products.Zuul import getFacade
+from Products.Zuul.interfaces import ICatalogTool
+
+
+@monkeypatch('Products.ZenModel.Device.Device')
+def getPingStatus(self):
+    if self.path()[0][6] == 'Linux':
```

Comment:
A path check isn't a good way to determine if this is a Linux device being monitoring with SSH. Since we don't have a specific type to check for, some deeper checking might be in order. Perhaps looking through self.getRRDTemplates() for COMMAND datasources with usessh enabled..

```python
def monitored_with_ssh(device):
    for template in device.getRRDTemplates():
        for datasource in template.datasources():
            if datasource.sourcetype == 'COMMAND' and datasource.usessh:
                return True

    return False
```